### PR TITLE
Support CLEAROS run BVT and Performance

### PR DESCRIPTION
**Pull request by LiliDeng**

### CODE REVIEW

File: Testscripts/Linux/BVT-VERIFY-UUID-FSTAB.py [hunk 1]
```diff
@@ -52,6 +52,14 @@ def RunTest():
         else:
             RunLog.info('root partition is not mounted using LABEL in dmesg.')
             ResultLog.info('FAIL')
+    elif DetectDistro()[0] == 'clear-linux-os':
```
Comment (ader1990) [inline Testscripts/Linux/BVT-VERIFY-UUID-FSTAB.py]: can you run pep8/flake8 on this script? these are python linters

File: Testscripts/Linux/azuremodules.py [hunk 2]
```diff
@@ -96,7 +96,10 @@ def DetectDistro():
     version = 'unknown'
```
Comment (ader1990) [inline Testscripts/Linux/azuremodules.py]: pep8/flake8 run

File: Testscripts/Linux/azuremodules.py [hunk 3]
```diff
@@ -96,7 +96,10 @@ def DetectDistro():
...
 
     RunLog.info("Detecting Distro ")
-    output = Run("cat /etc/*-release")
+    if os.path.isfile("/etc/*-release"):
```
[the remark this answers is not in the document]
Comment (LiliDeng) [reply Testscripts/Linux/azuremodules.py]: I don't understand this comments, could you please clarify more? thanks!

File: Testscripts/Linux/azuremodules.py [hunk 4]
```diff
@@ -159,11 +162,15 @@ def DetectLinuxDistro():
         return (True, "Debian")
     if os.path.isfile("/etc/SuSE-release"):
         return (True, "Suse")
+    if os.path.isfile("/usr/lib/os-release") and "clear-linux-os" in GetFileContents("/usr/lib/os-release"):
```
[the remark this answers is not in the document]
Comment (LiliDeng) [reply Testscripts/Linux/azuremodules.py]: @juhlee-microsoft inside which block?

File: Testscripts/Linux/azuremodules.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (juhlee-microsoft) [reply Testscripts/Linux/azuremodules.py]: I recall this comment and propose new one. GetDistro and DetectDistro are mostly duplicated. Many places GetDistro is used.

File: Testscripts/Linux/azuremodules.py (hunk 4 above)
[the remark this answers is not in the document]
Comment (juhlee-microsoft) [reply Testscripts/Linux/azuremodules.py]: I mis-read it . close this.

File: Testscripts/Linux/azuremodules.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (LiliDeng) [reply Testscripts/Linux/azuremodules.py]: @juhlee-microsoft DetectDistro is defined in py file, GetDistro  is defined in sh file. We can file a task to unify usage of get distro.

**Comment by LiliDeng** [comment]
@juhlee-microsoft @ader1990 Please help review again, thanks much!

**Comment by LiliDeng** [comment]
@ader1990 I just make sure my updated code, no errors when run pep8/flake8

File: Testscripts/Linux/azuremodules.py (hunk 2 above)
Comment (LiliDeng) [reply Testscripts/Linux/azuremodules.py]: @ader1990 I make sure my updated code without error, for remains we can have a task to fix all errors for existed py files.

**Comment by lizzha** [comment]
@ader1990 can you please sign off?

**Pull request merged**